lib/plugin.lazy.js

```diff
@@ -0,0 +1,110 @@
+import VueLib from 'vue'
+
+<% if (options.lazy.injectMock) { %>
```

Comment:
I absolutely hate templates in plugins as then both linting and type checking are not usable.
For nuxt-i18n I'm putting all options in a separate template and then importing those from the actual plugins. Of course you can't control if the code is inserted or not but typically control flows are enough.

I'm not saying that should be changed for this PR but I'll probably try that later if it's not.

Reply:
Afaik templates can & will be dropped for Nuxt v3 at least ;)